documentation/modules/post/windows/gather/credentials/thycotic_secretserver_dump.md

```diff
@@ -0,0 +1,102 @@
+This module exports and decrypts Secret Server credentials to a CSV file; it is intended as a
+post-exploitation module for Windows hosts with Delia/Thycotic Secret Server installed. Master
+Encryption Key (MEK) and associated IV values are decrypted from `encryption.config` using a 
+static key baked into the software. The module also supports parameter recovery for encryption
+configs configured with Windows DPAPI. An optional parameter "LOOT_ONLY" allows the encryption
+keys and encrypted database to be plundered for later offline decryption in situations where
+expedience is necessary.
+
+This module incorporates original research published by the authors of SecretServerSecretStealer,
+a PowerShell script designed to harvest Secret Server credentials. The GitHub repo for
+SecretStealer.ps1 includes tons of notes on the internals of Secret Server:
+
+https://github.com/denandz/SecretServerSecretStealer
+
+## Vulnerable Application
+This module should have no problem decrypting the database for versions 10.4 through 11.2, though
+it has only been tested against a system running version 11.2.  It is intended to be run after
+successfully exploiting a Windows host with the Delia/Thycotic Secret Server software installed.
+The module supports decryption of configuration files that have been protected by Windows DPAPI,
+but does not support extraction of any secrets if the system is configured with a Hardware Security
+Module (HSM).
+
+## Verification Steps
+This is a post module and requires a meterpreter session on the Microsoft Windows server host
+with a configured instance of Delia/Thycotic Secret Server installed.
+
+1. Start msfconsole
+2. Get session on Secret Server host via method of choice and background it
+3. Do: `use post/windows/gather/credentials/thycotic_secretserver_dump`
+4. Do: `set session <session>`
+5. Do: `dump`
+
+## Options
+
+### SESSION
+
+Which session to use, which can be viewed with `sessions -l`
+
+### LOOT_ONLY
+
+Boolean value that determines whether the module terminates after extracting the elements required
+to perform the decryption. If set to `true`, the module terminates after the MEK and database have
+been added to loot; if set to `false`, the module will perform decryption of the database using
+the extracted MEK values and add the resulting decrypted data to loot. This is a useful option to
+enable where rapid exfiltration is desirable. 
+
+## Scenarios
+Example run from meterpreter session on a Windows Server 2019 host running Secret Server 11.2:
```

Comment:
```suggestion
# Windows Server 2019 host running Secret Server 11.2
```

Reply:
I'm going to need to update the heck out of the docs regardless but will def. be less verbose when I do.